Sources/xcodeproj/Objects/Project/PBXProj.swift

```diff
@@ -212,16 +212,16 @@ extension PBXProj: Equatable {
 // MARK: - Writable
 
 extension PBXProj: Writable {
-    public func write(path: Path, override: Bool) throws {
-        try write(path: path, override: override, outputSettings: PBXOutputSettings())
+    public func write(path: Path, override: Bool) {
+        write(path: path, override: override, outputSettings: PBXOutputSettings())
     }
 
-    public func write(path: Path, override: Bool, outputSettings: PBXOutputSettings) throws {
+    public func write(path: Path, override: Bool, outputSettings: PBXOutputSettings) {
         let encoder = PBXProjEncoder(outputSettings: outputSettings)
-        let output = try encoder.encode(proj: self)
+        let output = encoder.encode(proj: self)
         if override, path.exists {
-            try path.delete()
+            try! path.delete()
```

Comment:
Same with the bang here, if a delete failed I wouldn't want my app to crash. Deletes fail, it's the nature of filesystems, eg. permissions may be wrong.